src/pudl_archiver/archivers/eia/eia176.py

```diff
@@ -41,7 +42,11 @@ async def get_items(self, url: str = items_url) -> list[str]:
     async def get_resources(self) -> ArchiveAwaitable:
         """Download EIA 176 resources."""
         items_list = await self.get_items(url=self.items_url)
-        for year in range(1997, 2023):
+        datasets_list = await self.get_datasets(url=self.base_url, form=self.form)
```

Comment:
Should this line still be referring to `self.base_url` which has been removed above?

Reply:
`base_url` is defined in the `EiaNGQVArchiver` class, it is the correct link here. I've updated the docstring to clarify.